src/modules/StudyManager/components/List/List/components/Table/presenter.jsx

```diff
@@ -56,6 +62,18 @@ function CellName(props) {
   );
 }
 
+function RoleCell({ value, index }) {
+  const classes = new BEMHelper('cell-name');
+
+  return (
+    <div {...classes()}>
+      <span title={value} key={index + 'table-cell-text-span'} {...classes('title')}>
```

Comment:
Putting index at the start of the key rather than at the end looks a bit exotic to me. Any specific reason we are doing it this way?